cmd/upgrade/roles/cmd.go

```diff
@@ -668,12 +663,21 @@ func buildAccountRoleCommandsFromCluster(
 			}
 			_, err = awsClient.IsPolicyExists(policyARN)
 			hasPolicy := err == nil
+			policyAttached := false
```

Comment:
Can policyAttached by renamed to something like
RequiredRolePolicyAttached?
The difference between HasPolicy and PolicyAttached isn't clear for me. 😢

Reply:
hasPolicy is renamed to policyExists